### ISSUE TITLE
No pokemon on mobile

### ISSUE THREAD

**Issue by Clowning** [AHAAAAAAA/PokemonGo-Map#1628]
There are no pokemon on map if i use my mobile phone, but on my desktop is ok

I have an iPhone

**Comment by thomasmoors** [comment]
You could install chrome as temp fix

**Comment by ahknight** [comment]
Works on my iPhone in Safari just fine. It'd be good to list some steps.  There's a delay on updates if you changed your location after loading it on the phone, for instance.

**Comment by ahknight** [comment]
@thomasmoors Nope, Chrome on the iPhone uses the same engine as Safari.  iOS limitation/restriction.

**Comment by thomasmoors** [comment]
@ahknight Ah that is too bad. Good luck guys

**Comment by Clowning** [comment]
Ok hum... The website works if i add the website on my app screen x) ! Good news ! Maybe you should to add this on the readme / wiki !

**Comment by ahknight** [comment]
That starts a new instance of the web engine (same as a new tab). My guess is the one instance of Safari's WebView got borked or hit some bad JS and stopped processing.  A new tab should have the same effect as the home icon.

**Comment by Clowning** [comment]
Oh ok good to know ! Thanks :) !

**Comment by PoorNewb23** [comment]
Uhm, we have same problem on newest and greatest mobile Chrome.
Reproduction steps:
-add 8+ filters and notifications.
-after initial seeding, pokemons stops updating.
Ones for which notifications are enabled willstob being clickable.
Reloading the page or opening in new tab does not help.

**Comment by Creat1v** [comment]
Same issue with iPhone 5 iOS 8.4. Just a white screen without informations.
Tried with Safari, Chrome and Firefox.
Works on iPhone 5s iOS 9.x.

**Comment by PoorNewb23** [comment]
Found source - it's notifications. they do not work and cause JS to fail.
